Gemfile.lock

```diff
@@ -2,35 +2,49 @@ GEM
   remote: https://rubygems.org/
   specs:
     ast (2.4.2)
+    base64 (0.1.1)
     json (2.6.3)
-    language_server-protocol (3.17.0.2)
-    parallel (1.22.1)
-    parser (3.1.3.0)
+    language_server-protocol (3.17.0.3)
+    lint_roller (1.1.0)
+    parallel (1.23.0)
+    parser (3.2.2.4)
       ast (~> 2.4.1)
+      racc
+    racc (1.7.2)
     rainbow (3.1.1)
-    regexp_parser (2.6.1)
-    rexml (3.2.5)
-    rubocop (1.40.0)
+    regexp_parser (2.8.2)
+    rexml (3.2.6)
+    rubocop (1.56.4)
+      base64 (~> 0.1.1)
       json (~> 2.3)
+      language_server-protocol (>= 3.17.0)
       parallel (~> 1.10)
-      parser (>= 3.1.2.1)
+      parser (>= 3.2.2.3)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
       rexml (>= 3.2.5, < 4.0)
-      rubocop-ast (>= 1.23.0, < 2.0)
+      rubocop-ast (>= 1.28.1, < 2.0)
       ruby-progressbar (~> 1.7)
-      unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.24.0)
-      parser (>= 3.1.1.0)
-    rubocop-performance (1.15.1)
+      unicode-display_width (>= 2.4.0, < 3.0)
+    rubocop-ast (1.30.0)
+      parser (>= 3.2.1.0)
+    rubocop-performance (1.19.1)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
-    ruby-progressbar (1.11.0)
-    standard (1.20.0)
+    ruby-progressbar (1.13.0)
+    standard (1.31.2)
       language_server-protocol (~> 3.17.0.2)
-      rubocop (= 1.40.0)
-      rubocop-performance (= 1.15.1)
-    unicode-display_width (2.3.0)
+      lint_roller (~> 1.0)
+      rubocop (~> 1.56.4)
+      standard-custom (~> 1.0.0)
+      standard-performance (~> 1.2)
+    standard-custom (1.0.2)
+      lint_roller (~> 1.0)
+      rubocop (~> 1.50)
+    standard-performance (1.2.1)
+      lint_roller (~> 1.1)
+      rubocop-performance (~> 1.19.1)
+    unicode-display_width (2.5.0)
```

Comment:
I did `bundle update standard` so that we can get onto a newer version of the linter. A bunch of other stuff was upgraded along with it, but this lock file is really only used in CI so I don't think it's anything we need to worry too much about.